src/SassBinary.php

```diff
@@ -16,16 +16,18 @@
 
 class SassBinary
 {
-    private const VERSION = '1.63.6';
+    private const DEFAULT_VERSION = '1.63.6';
     private HttpClientInterface $httpClient;
 
     public function __construct(
         private string $binaryDownloadDir,
         private ?string $binaryPath = null,
+        private ?string $binaryVersion = null,
```

Comment:
I was considering the BC break, but figured the amount or projects using the SassBinary directly will not yet be that big, and due to the next argument not being a string, it will be impossible to not notice this if you're building some sass and you've configured the `$output` argument yourself.

Happy to do whatever you guys feel is most suitable in this case.

Reply:
Yeah, we probably can neglect it in this spot, and you have reasonable reasons why. We can also mention a possible BC break in the release description just in case